### PR TITLE
Restore rank model versions on causatives/verified pages

### LINKED ISSUE #3933: Rank model version missing from rank scores on causatives page and exports
We used to have them, so guessing they are mostly lost in refactoring.
Screenshot 2023-05-09 at 16 14 46

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3933 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Check the Verified/Causatives pages after marking some vars as verified/causative

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/institutes/controllers.py [hunk 1]
```diff
@@ -110,6 +91,29 @@ def verified_stats(institute_id, verified_vars):
...
+            variant_obj, variant_genes
+        )  # required to display cDNA and protein change
+    variant_obj.update(predictions(variant_genes))
+    variant_obj["case_obj"] = {
+        "display_name": case_obj["display_name"],
+        "individuals": case_obj["individuals"],
+        "status": case_obj.get("status"),
+        "partial_causatives": case_obj.get("partial_causatives", []),
+        "rank_model_version": case_obj.get("rank_model_version"),
+        "sv_rank_model_version": case_obj.get("sv_rank_model_version"),
```
Comment (dnil) [inline scout/server/blueprints/institutes/controllers.py:110]: 👍

File: scout/server/blueprints/institutes/controllers.py [hunk 2]
```diff
@@ -110,6 +91,29 @@ def verified_stats(institute_id, verified_vars):
     return true_pos, false_pos, n_validations_ordered - (true_pos + false_pos)
 
 
+def decorate_institute_variant(variant_obj: dict) -> Optional[dict]:
```
Comment (dnil) [inline scout/server/blueprints/institutes/controllers.py:92]: 👍

**Comment by northwestwitch** [comment]
Local test 🆗 

image

**Review by dnil — APPROVED** [review]
Nice!

**Pull request merged**